src/pages/DashboardPage.tsx

```diff
@@ -112,6 +181,29 @@ const DashboardPage: React.FC = () => {
                 <MetricName>questions</MetricName>
               </Metric>
             )}
+
+            {templateFilters.map((filter, index) => (
+              <FilteredMetric
+                onCreate={createTemplate}
+                onRemove={removeTemplate}
+                onEdit={editTemplateWithName}
+                key={index + templateFilters.length}
```

Comment:
Can't we do this now?

```suggestion
                key={filter.id}
```